### PR TITLE
Stop adding default desktop assembly reference repeatedly.

### PR DESCRIPTION
**Pull request by troydai**
Issue: #2344

---
This change is [Reviewable](https://reviewable.io/reviews/dotnet/cli/2363)

### CODE REVIEW

**Comment by troydai** [comment]
Investigating the CI failures.

**Comment by troydai** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build please.

**Comment by troydai** [comment]
@dotnet-bot CentOS7.1 x64 Debug Build

**Comment by troydai** [comment]
CentOS7.1 x64 Debug Build please

**Comment by troydai** [comment]
@dotnet-bot CentOS7.1 x64 Debug Build please

**Comment by troydai** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build please

**Comment by troydai** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build please

**Comment by troydai** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build please

**Comment by Sridhar-MS** [comment]
@mmitche Some of the CentOS CIs are failing with error

> java.io.IOException: Failed to mkdirs: /mnt/resource/j/workspace/dotnet_cli/rel_1.0.0/debug_centos7.1_x64_prtest

http://dotnet-ci.cloudapp.net/job/dotnet_cli/job/rel_1.0.0/job/debug_centos7.1_x64_prtest/961/console

Any idea?

**Comment by mmitche** [comment]
lt's that azure either didn't attach the disk or not soon enough.  Deleted it.

@dotnet-bot test CentOS7.1 x64 Debug Build

**Comment by troydai** [comment]
Thanks @mmitche all passed.

**Comment by troydai** [comment]
@anurse can you review this for me?

**Comment by anurse** [comment]
:shipit:

File: TestAssets/TestProjects/TestMscorlibReference/project.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "version": "1.0.0-*",
+  "dependencies": {},
+  "frameworks": {
+    "netstandardapp1.5": {
+      "dependencies": {
+        "NETStandard.Library": "1.5.0-rc2-23931"
```
Comment (weshaggard) [inline TestAssets/TestProjects/TestMscorlibReference/project.json]: These version should be 24008 now.

Comment (anurse) [inline TestAssets/TestProjects/TestMscorlibReference/project.json]: Good catch. @troydai make these consistent with whatever everything else is using.

Comment (weshaggard) [inline TestAssets/TestProjects/TestMscorlibReference/project.json]: Actually I know as part of https://github.com/dotnet/cli/pull/2331 @eerhardt is also removing netstandardapp references and instead switching to netcoreapp1.0 which might be necessary here as well.

Comment (troydai) [inline TestAssets/TestProjects/TestMscorlibReference/project.json]: I'll do that. Thanks for reminding.

**Comment by troydai** [comment]
@eerhardt I will leave the update test projects framework to `netcoreapp1.0` to you since my PR is based on the commit without tests target tests changed. 

Updated the version to `24008` and rebase on latest rel/1.0.0. The .NET core shiproom has approved this change, I'll wait till all CI build pass and then merge.

**Comment by troydai** [comment]
Oh, it looks like @eerhardt's changes are merged an hour ago. I'll update the target framework then.

**Comment by troydai** [comment]
Look at the #2331 again the change of target framework for test projects is not required. The tests projects here are only for desktop CLR scenarios.

**Pull request merged**